ethcore/src/machine.rs

```diff
@@ -60,6 +69,39 @@ pub struct EthashExtensions {
 	pub dao_hardfork_beneficiary: Address,
 	/// DAO hard-fork DAO accounts list (L)
 	pub dao_hardfork_accounts: Vec<Address>,
+	/// Number of first block wehre Casper rules begin.
+	pub hybrid_casper_transition: u64,
```

Comment:
Given that we want to use casper in other deployments, we should not implement it as an ethash extension.

Reply:
s/wehre/where

Reply:
Will handle the refactoring soon, once all the protocol stuff is done. :)